src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -290,9 +292,28 @@ private static Response mergeNativeImpTrackers(Response response, List<EventTrac
                 .build();
     }
 
-    private static BidType getBidType(String impId, List<Imp> imps) {
+    private static BidType getBidType(Bid bid, List<Imp> imps) {
+        Integer mType = bid.getMtype();
```

Comment:
According to project style, we prefer to use final keyword on variables declaration